server/models.py

```diff
@@ -0,0 +1,72 @@
+from app import db
+from sqlalchemy.sql import func
+from sqlalchemy import ForeignKey
+
+class contest_user(db.Model):
+    __tablename__ = 'contest_user'
+
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String())
+    password = db.Column(db.String())
+    createdTime = db.Column(db.DateTime, default=func.utcnow())
```

Comment:
would be best if these attributes are snake_case

```suggestion
    created_time = db.Column(db.DateTime, default=func.utcnow())
```